src/garage/np/policies/policy_utils.py

```diff
@@ -0,0 +1,40 @@
+"""Module for utility functions commonly used by policies."""
```

Comment:
please do not introduce new `_utils` modules. This should go in garage.np and src/garage/np/_functions.py